app/src/main/java/ru/prometeydev/movie/ui/moviesdetails/MoviesDetailsFragment.kt

```diff
@@ -0,0 +1,122 @@
+package ru.prometeydev.movie.ui.moviesdetails
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.ImageView
+import android.widget.RatingBar
+import android.widget.TextView
+import androidx.core.content.ContextCompat
+import androidx.recyclerview.widget.DividerItemDecoration
+import androidx.recyclerview.widget.LinearLayoutManager
+import androidx.recyclerview.widget.RecyclerView
+import coil.load
+import ru.prometeydev.movie.R
+import ru.prometeydev.movie.data.adapters.ActorsAdapter
+import ru.prometeydev.movie.data.Movie
+import ru.prometeydev.movie.data.adapters.calculateStarsCount
+
+class MoviesDetailsFragment : MoviesDetailsNavigable() {
+
+    private var recycler: RecyclerView? = null
+
+    private var movie: Movie? = null
+
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View = inflater.inflate(R.layout.fragment_movies_details, container, false)
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+
+        loadData()
+        movie?.let { setupViews(view, it) }
+        viewModel.actorsListEmptyState.observe(this.viewLifecycleOwner, this::showMessageIfNeeded)
```

Comment:
Why do you need view model if data already in activity?

Reply:
I will remove this vie model from here, it is really not needed.